org.eclipse.draw2d-feature/feature.xml

```diff
@@ -15,7 +17,7 @@
       version="3.18.0.qualifier"
       provider-name="%providerName"
       license-feature="org.eclipse.license"
-      license-feature-version="1.0.1">
+      license-feature-version="0.0.0">
```

Comment:
Wouldn't it make sense to instead set the version to 2.0.1? Otherwise we'd automatically opt-in, in case the Eclipse ever decides to migrate to a newer version again.

Reply:
I was not sure. I checked what the Eclipse Platform has as value and did the same.

Reply:
This is what I thought so too. And if the platform does it we can not be that wrong.